manipulation/src/functTest/java/org/jboss/gm/manipulation/SimpleProjectWithSpringDMAndMavenPluginsFunctionalTest.java

```diff
@@ -48,17 +50,27 @@ public void ensureProperPomGenerated() throws IOException, URISyntaxException, X
                 .forwardOutput()
                 .withPluginClasspath()
                 .build();
-        assertThat(buildResult.task(":install").getOutcome()).isEqualTo(TaskOutcome.SUCCESS);
+        assertThat(Objects.requireNonNull(buildResult.task(":install")).getOutcome()).isEqualTo(TaskOutcome.SUCCESS);
```

Comment:
Is this due to warnings in IntelliJ ? - that the code there may throw `NullPointerException`? However, Objects.requireNonNull just throws NullPointerException so I'm not sure there is much benefit gained.

Reply:
Yes, this was just to get rid of Intellij warnings. I should probably use assertion instead, will fix.